### DISCUSSION TITLE
Private RSS sources

### DISCUSSION THREAD

**Discussion by xwillq** [dailydotdev/daily#778] [Feature Requests]
## Description
Content guidelines severely restrict what we can read on Daily.dev. For example, if I like some personal blog, I cannot use Daily.dev to read it and have to get RSS reader or visit blog to check for new posts. Same goes for sources in other languages or blogs that publish very rarely.

Adding functionality to add sources that will be visible only to particular user will solve that problem and allow users to unify most of their favorite blogs in one place and use the same reading history and bookmarks list for all of them.

## Posible implementation
1. Add source subscriptions (Feature request #715)
2. Add flag `private` to sources
3. Add functionality to add sources without moderation, but make them private
4. Subscribe user to private source that they added

**Comment by idoshamun** [comment]
Hi @xwillq,

We used to have this feature as part of our premium subscription (that is no longer available).
I understand your concern though, and we have ideas on how to solve it.
Might take some time to address it properly.

**Reply by TekGadgt** [reply]
Would it make sense to potentially allow something like this in squads?  I was just setting one up for some friends and was hoping something like this existed so we could all get updates from the same sources regularly, amongst our one off posts.  Would also potentially be useful for squads centered around a particular creator or technology that posts frequently.  Just a thought, no issue posting things on my own for now.

**Reply by idoshamun** [reply]
It definitely makes sense. There's a technical challenge though. Adding a source has an overhead and can be quite expensive, that's why we're not rushing to do it

**Reply by TekGadgt** [reply]
That makes total sense!  Just something I thought of as I was starting to dig more into using the platform regularly, and rather than start up a whole new discussion, found this one where something similar had already been mentioned.
